setup.py

```diff
@@ -94,7 +94,12 @@ def fetch_readme():
 
 def get_version():
     with open('version.txt') as f:
-        return f.read().strip()
+        version = f.read().strip()
+        if build_cuda_ext:
+            torch_version = ''.join(torch.__version__.split('.')[:2])
```

Comment:
[:2] is hard coding...
from packaging import version
use this lib to detect version number.

Reply:
OK, I will test it and create a PR later.

Reply:
`packaging` is not in standard library. If users want to install from source, they must install `pacaking` first. I think it's a trouble for users, so I think we should not use `packaging` now.